planning/behavior_velocity_planner/src/scene_module/virtual_traffic_light/scene.cpp

```diff
@@ -399,10 +410,10 @@ bool VirtualTrafficLightModule::modifyPathVelocity(
     return true;
   }
 
-  // Stop at stop_line if state is timeout
-  if (isBeforeStopLine() || planner_param_.check_timeout_after_stop_line) {
+  // Stop at stop_line if state is timeout before stop_line
+  if (isBeforeStopLine()) {
     if (isStateTimeout(*virtual_traffic_light_state)) {
-      RCLCPP_DEBUG(logger_, "state is timeout");
+      RCLCPP_DEBUG(logger_, "state is timeout before stop line");
```

Comment:
Will you add printing `instrument_id` in another PR?

Reply:
I removed it but I will send another PR for it.

Reply:
I send another PR. https://github.com/autowarefoundation/autoware.universe/pull/328